src/pages/arbeidsgiver/[narmestelederid]/[oppfolgingsdialogId]/arbeidsoppgaver.tsx

```diff
@@ -0,0 +1,37 @@
+import { useChosenAktivOppfolgingsplanAG } from "api/queries/arbeidsgiver/oppfolgingsplanerQueriesAG";
+
+import { NextPage } from "next";
+import React from "react";
+import {
+  OppfolgingsplanPageAG,
+  Page,
+} from "components/blocks/wrappers/OppfolgingsplanPageAG";
+import { LagredeArbeidsoppgaver } from "components/arbeidsoppgaver/LagredeArbeidsoppgaver";
+import { beskyttetSideUtenProps } from "../../../../auth/beskyttetSide";
+import { NyArbeidsoppgaveAG } from "../../../../components/arbeidsoppgaver/NyArbeidsoppgaveAG";
+
+const Arbeidsoppgaver: NextPage = () => {
+  const aktivPlan = useChosenAktivOppfolgingsplanAG();
+  const arbeidstakerFnr = aktivPlan?.arbeidstaker.fnr;
+  if (!arbeidstakerFnr) return null;
```

Comment:
Hvordan vil dette se ut for bruker, dersom denne blir null? Helt tom side uten noe?

Reply:
..dersom dette faktisk kan skje (usikker?), så bør vi kanskje ha en feilmelding om det

Reply:
Stemmer at det blir tom side. Denne var kopiert fra SM variant, så fikser jeg begge slikt du foreslår 👍🏻